src/keploy.ts

```diff
@@ -1,4 +1,5 @@
 import HttpClient, { Request } from "./client";
+import { transformToSnakeCase } from "./util/util";
```

Comment:
Hey can you remove this nested `util`. Just create a file `src/utils.ts`

Reply:
This is now done as of 41056c1.